frontend/src/app/components/containers/complaints/complaint-list.tsx

```diff
@@ -101,26 +102,39 @@ export const ComplaintList: FC<Props> = ({ type, searchQuery }) => {
 
   const totalComplaints = useAppSelector(selectTotalComplaintsByType(type));
   const defaultPageSize = useAppSelector(selectDefaultPageSize);
+  const storedSearchParams = useAppSelector(selectComplaintSearchParameters);
+  const { sortColumn, sortOrder } = storedSearchParams;
+  const freshSearch = Object.keys(storedSearchParams).length === 2 ? true : false;
```

Comment:
I think the sonar comment is accurate here - could be simplified to:

const freshSearch = Object.keys(storedSearchParams).length === 2

Reply:
Ahh good catch